app/src/renderer/components/governance/ModalDeposit.vue

```diff
@@ -138,48 +96,38 @@ export default {
         required,
         isInteger,
         between: between(1, this.balance)
-      },
-      password: {
-        required
       }
     }
   },
   methods: {
-    close() {
-      this.$emit(`update:showModalDeposit`, false)
+    open() {
+      this.$refs.actionModal.open()
     },
-    async validateForm() {
+    validateForm() {
```

Comment:
do we still need this in the individual action modals?

Reply:
yes each modal has an individual form it needs to validate. each form validation could be completely different so a parent form performs it's own validation (triggered by the ActionModal).